### PR TITLE
feat: Add profile picture upload field and display avatar image on Account Page

### PR DESCRIPTION
**Pull request by kkumar-gcc**
This PR introduces the necessary UI components to implement the profile picture upload functionality on the Account Page. The changes include:
- Designing and implementing the UI components for the profile picture upload field.
- Displaying the avatar image on the Account Page.
- Implementing the functionality to open the image upload dialog when clicking on the avatar image.
- Storing the uploaded image in the back-end to associate it with the user's account.

Fixes:  #49

### CODE REVIEW

File: components/interfaces/Account/ProfileImageUpload.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <div className="relative w-36 h-36 mb-4">
+          {formik.values.image ? (
+            <img
+              src={formik.values.image}
+              className="w-36 h-36 rounded-full absolute m-auto shadow"
+              alt=""
+            />
+          ) : (
+            <img
+              src="https://www.pngkey.com/png/detail/121-1219231_user-default-profile.png"
```
Comment (devkiran) [inline components/interfaces/Account/ProfileImageUpload.tsx:33]: Don't use the external image link. Please download it and put in locally.

File: components/interfaces/Account/ProfileImageUpload.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              className="w-36 h-36 rounded-full absolute m-auto shadow"
+              alt="Default Image"
+            />
+          )}
+        </div>
+        <div className="">
```
Comment (devkiran) [inline components/interfaces/Account/ProfileImageUpload.tsx:39]: Remove `className=""`

File: components/interfaces/Account/UpdateAccount.tsx [hunk 3]
```diff
@@ -21,10 +38,12 @@ const UpdateAccount = ({ user }: { user: User }) => {
     initialValues: {
       name: user.name,
       email: user.email,
+      image: user.image
     },
     validationSchema: schema,
     onSubmit: async (values) => {
       try {
+        console.log(values)
```
Comment (devkiran) [inline components/interfaces/Account/UpdateAccount.tsx:46]: Remove `console.log(values)`

File: pages/api/users.ts [hunk 4]
```diff
@@ -20,17 +20,17 @@ export default async function handler(
...
-  const { name } = req.body;
+  const { name, image } = req.body;
 
   const session = await getSession(req, res);
 
   if (!session) {
     return res.status(401).json({ error: { message: 'Unauthorized' } });
   }
-
+  console.log(name, image);
```
Comment (devkiran) [inline pages/api/users.ts:30]: Remove `console.log(name, image);`

File: components/interfaces/Account/ProfileImageUpload.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import React, { ChangeEvent, useRef } from 'react';
```
Comment (devkiran) [inline components/interfaces/Account/ProfileImageUpload.tsx:1]: Can you add the translation keys for new texts?

**Review by devkiran — CHANGES_REQUESTED** [review]
The avatar upload looks great! Here are are a few things I noticed

**Review by devkiran — APPROVED** [review]

**Comment by NathanTarbert** [comment]
Thanks @kkumar-gcc, this is a great step forward for UI customization.

**Pull request merged**